src/biogeochem/CropType.F90

```diff
@@ -335,7 +342,11 @@ subroutine InitAccBuffer (this, bounds)
 
     !---------------------------------------------------------------------
 
-    call init_accum_field (name='GDDPLANT', units='K', &
+    call init_accum_field (name='HUI', units='K', &
```

Comment:
- [ ] Although this rename is nice, I think we'll need to do something for the sake of backwards compatibility with existing restart files, because each accumulation variable has a few restart variables associated with it. If this were just a matter of changing a couple of history fields, I don't think this would be important, but my impression is that formerly-GDDPLANT-now-HUI impacts the evolution of the model, right? Options I can see are:
  1. Keep the original name for this accumulation field, even though it means that the accumulation field will be named differently from the main variable name.
  2. Back out this entire rename, sticking with GDDPLANT throughout the code. (You could still add a second variable like you do in this PR.)
  3. Keep your full rename, but add some backwards compatibility capabilities in accumulMod. I think this might not be too hard (famous last words): we have a mechanism for specifying old variable names in calls to restartvar by giving a colon-delimited string like "current_varname:old_varname". So I think this would consist of:
    - Add an optional argument to `init_accum_field` like `old_name` (in principle we could allow an array of old names, but I'm thinking we shouldn't go too crazy here)
    - In `accumulRest`, if `accum(nf)%old_name` is set, we'll need to modify the restart var names accordingly. For example, the `_VALUE` name will become: `varname = trim(accum(nf)%name) // '_VALUE' // ':' // trim(accum(nf)%old_name) // '_VALUE'`, and similarly for the `_NSTEPS` and `_PERIOD` variable names.
    - Testing: I *think* the above is all that's needed, but you should test to ensure that:
      - The actual variable names on the restart file are correct (the new names)
      - If you do an `ncdump -h` on a restart file, you should see a piece of metadata on each field called `varnames_on_old_files`. For variables with a backwards-compatibility field like this, you'll see something like: `LIQCAN:varnames_on_old_files = "LIQCAN:H2OCAN"`. You should check to make sure this appears correctly. (This piece of metadata is read by the code to implement the backwards compatibility if you're interpolating initial conditions via init_interp.)
      - A test against baselines should be bit-for-bit (I suspect this is *not* currently the case for this branch because it doesn't have this backwards compatibility mechanism in place).